src/redux/books/books.js

```diff
@@ -0,0 +1,19 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export const addBook = () => ({
+  type: ADD_BOOK,
+});
+
+export const removeBook = () => ({
+  type: REMOVE_BOOK,
+});
```

Comment:
- [ ] Same here, Kindly Pass the `id` of the book so the reducer can use the id to remove the book when the `removeBook `action is dispatched.

Reply:
done